commit 4a212f707b6bd9b6014f28914f2e9e6e1bfb9015
Merge: 74060f1 24e70c9
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Thu Jun 9 20:56:07 2016 -0700

    Merge pull request #3993 from jbrazio/feature/update-travis
    
    Update travis to test for ULTRA and DOG LCD options

diff --git a/.travis.yml b/.travis.yml
index a493d9c..2464091 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -132,6 +132,17 @@ script:
   #
   ### LCDS ###
   #
+  # ULTRA_LCD
+  #
+  - restore_configs
+  - opt_enable ULTRA_LCD
+  - build_marlin
+  #
+  # DOGLCD
+  #
+  - restore_configs
+  - opt_enable DOGLCD
+  - build_marlin
   #
   # ULTIMAKERCONTROLLER
   #
